### DISCUSSION TITLE
Github Files integration is not hiding source code title & link

### DISCUSSION THREAD

**Discussion by Boburmirzo** [GitbookIO/community#576] [0. General bugs and questions]
Hi,

I enabled GitHub Files integration and external source code is shown on the Gitbook page. However, the feature to hide the title & link is not working when I clicked on it. 

Could you check why, please?

It is easy to reproduce. Just put embed link from the GitHub code and it shows now title and link. See the screenshot:

![Screenshot 2024-04-11 at 1 22 31 PM](https://github.com/GitbookIO/community/assets/14247607/c7d7543d-a327-41c3-9bec-52296367d21f)

**Comment by aikoii** [comment]
@Boburmirzo thanks for pointing that out! We'll review this 👍
